### ISSUE TITLE
not working properly on phpstrom2016.1

### ISSUE THREAD

**Issue by avastamin** [daylerees/colour-schemes#201]
![screen shot 2016-05-28 at 23 16 35](https://cloud.githubusercontent.com/assets/1988935/15630023/6718e298-252a-11e6-9ac6-a036a1f9c7e9.png)

Color on top section not changing.

**Comment by normkatz** [comment]
Almost all of the daylerees themes for JetBrains do not appear correctly on the latest PHPStorm 2016.1.  The keywords like class and function appear Orange in php files even though they look great on your preview site.  I tried 3 of the contrast schemes including darkside-contrast, frontier-contrast, and zacks-contrast.  All had the same problem.  Other colors for these keywords appear different for some of the built-in themes.  I'm guessing jetbrains changed the tags.

**Comment by daylerees** [comment]
Hey guys. The top section of the editor is provided by the jetbrains theme for the PHPStorm UI (not mine). It won't change colour to match the colour scheme, sadly. I'll consider generating colour variants of it at some point.

@normkatz the example themes were built around Sublime Text, which was the first editor that I had themed. Sublime + PHPStorm have very different rendering engines, and I've found it near impossible to make them match. The PHPStorm variants are the best that I could manage with its theme description.

However, I suggest that you both checkout the following repo: https://github.com/daylerees/material-peacock - which is the best attempt of PHPStorm theming that I've had to date.

Thanks!
